test/unit/testtest_spec.lua

```diff
@@ -0,0 +1,19 @@
+local helpers = require('test.unit.helpers')(after_each)
+local assert = require('luassert')
+
+local itp = helpers.gen_itp(it)
+
+local sc = helpers.sc
+
+-- All of the below tests must fail. Check how exactly they fail.
+if os.getenv('NVIM_TEST_RUN_TESTTEST') ~= '1' then
+  return
+end
```

Comment:
It needs to be checked when test implementation changes. I was thinking about making that actual test (run always), but very unsure how I could do that, best guess is running busted directly from somewhere with some switches making the output more or less stable and passing the whole thing through sed to remove what is always not stable (times, some line numbers).

I do not like how the result sounds, so you do not see it being tested.